DXMainClient/DXGUI/Multiplayer/CnCNet/ChoiceNotificationBox.cs

```diff
@@ -0,0 +1,193 @@
+﻿using ClientCore;
+using ClientGUI;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Rampastring.XNAUI;
+using Rampastring.XNAUI.XNAControls;
+using System;
+
+namespace DTAClient.DXGUI.Multiplayer.CnCNet
+{
+    /// <summary>
+    /// A box that allows users to make a choice,
+    /// top-left of the game window.
+    /// </summary>
+    public class ChoiceNotificationBox : XNAPanel
+    {
+        const double DOWN_TIME_WAIT_SECONDS = 4.0;
+        const double DOWN_MOVEMENT_RATE = 2.0;
+        const double UP_MOVEMENT_RATE = 2.0;
```

Comment:
Mark these constants and private fields as explicitly private. While not all code in the client does it yet because there's a lot of outdated code around, it's good to apply higher standards to new code.

Reply:
Done.